src/Blocks/BlockDeadBush.h

```diff
@@ -42,6 +42,21 @@ class cBlockDeadBushHandler :
 		UNUSED(a_Meta);
 		return 0;
 	}
+
+	virtual void OnDestroyedByPlayer(cChunkInterface & a_ChunkInterface, cWorldInterface & a_WorldInterface, cPlayer * a_Player, int a_BlockX, int a_BlockY, int a_BlockZ) override
```

Comment:
We have too many similar methods to override, and then it leads to situations like this. It seems like the `DropBlock` method would be preferred for this indeed.